std/traits.d

```diff
@@ -1549,30 +1542,29 @@ unittest
         return !b;
     }
     static assert(hasFunctionAttributes!(myFunc!bool, "@safe", "pure", "@nogc", "nothrow"));
-    static assert(!hasFunctionAttributes!(myFunc!bool, "@trusted"));
+    static assert(!hasFunctionAttributes!(myFunc!bool, "shared"));
```

Comment:
this change is just to make the public `unittest` a bit more interesting and not use `@trusted` twice.